lib/ui/process_ui/new_process.dart

```diff
@@ -270,6 +270,10 @@ class _NewProcessState extends State<NewProcess> {
           isExceptionPresent = true;
           return isExceptionPresent;
         }
+        if(x.title=="Exception"){
```

Comment:
we should update the logic break the loop if exception is present and then return from method rather than have return statement after each if